File: ejercicios-js/ejercicio7.js
// Lista de palabras
const palabras = ["hola", "mundo", "javascript", "programación", "código"];

// Selecciona una palabra al azar
const palabraSeleccionada = palabras[Math.floor(Math.random() * palabras.length)];

// Invierte la palabra
const palabraInvertida = palabraSeleccionada.split("").reverse().join("");

// Muestra el resultado
console.log(`Palabra seleccionada: ${palabraSeleccionada}`);
console.log(`Palabra invertida: ${palabraInvertida}`);

File: ejercicios-py/ejercicio7.py
def invertir_palabra(palabra):
    return palabra[::-1]

# Definir una lista de palabras
palabras = ["hola", "mundo", "python", "programación"]

# Invertir cada palabra y devolver al revés
inversas = [invertir_palabra(p) for p in palabras]
print(inversas)